### ISSUE TITLE
Ergänze SkoHub-Link auf Systematikseiten

### ISSUE THREAD

**Issue by acka47** [hbz/nwbib#659]
Letzte Woche im Treffen mit @hmetzner besprochen (siehe  [Notizen](https://service-wiki.hbz-nrw.de/x/LwBdPw)):

### Anforderung

Auf `/subjects` und `/spatial` soll ein Link auf die jeweilge Repräsentation des Vokabulars mit SkoHub Vocabs zeigen.

### Umsetzung

Wir passen den jetzigen SKOS-Datei-Link an wie folgt.

Sachsystematik:

> SKOS-Version ([HTML](https://skohub.io/hbz/lobid-vocabs/heads/master/nwbib.de/subjects.html) , [Turtle](https://nwbib.de/subjects.ttl))

Raumsystematik:

> SKOS-Version ([HTML](https://skohub.io/hbz/lobid-vocabs/heads/master/nwbib.de/spatial.html) , [Turtle](https://nwbib.de/spatial.ttl))

**Comment by HMetzner** [comment]
Nur eine kurze Ergänzung von meiner Seite: dass der SkoHub-Link auch auf der Raumsystematik-Seite eingefügt wird, hatte ich nicht eingebracht, aber es wäre konsistent. An der Skos-Datei der Raumsystematik habe ich nichts geändert/ergänzt.

**Comment by acka47** [comment]
> dass der SkoHub-Link auch auf der Raumsystematik-Seite eingefügt wird, hatte ich nicht eingebracht, aber es wäre konsistent

Ja, genau, das war nicht Thema im Treffen, ich habe es aber der Konstenz wegen ergänzt.

**Comment by acka47** [comment]
@HMetzner kannst du bitte mal schauen, ob das für dich so passt mit den SkoHub-Links? -> https://test.nwbib.de/subjects

**Comment by HMetzner** [comment]
@acka47 @fsteeg vielen Dank für die Umsetzung! Das funktioniert gut. Könnte man dort, wo es zwei Verlinkungen gibt (Bsp. [532050 Ärzte. Heilberufe](https://test.nwbib.de/subjects#N532050) noch in Klammern und vielleicht hellgrau ergänzen? Damit man direkt weiß, welcher Link auf welchen Eintrag verweist?
Eine zweite Sache: Es könnte von Bearbeiter*innen die Rückmeldung kommen, dass das Link-Symbol direkt neben dem Kopier-Symbol unübersichtlich und bei der Bearbeitung hinderlich ist. Adrian, ich meine, wir hatten im Gespräch überlegt den Kasten alternativ mit den Notationen zu verlinken, oder? Dass der Kasten erscheint, wenn man auf die Wörter "Ärzte,Heilberufe" klickt. Wäre das möglich? Ansonsten könnte ich erstmal eine Feedback-Runde in Münster und Düsseldorf machen und schauen, ob die Bearbeitung auch so funktioniert.

**Comment by HMetzner** [comment]
Super, vielen Dank!
